src/Notes/WC_Admin_Notes.php

```diff
@@ -24,6 +24,7 @@ class WC_Admin_Notes {
 	 */
 	public static function init() {
 		add_action( 'admin_init', array( __CLASS__, 'schedule_unsnooze_notes' ) );
+		add_action( 'admin_init', array( __CLASS__, 'possibly_delete_survey_notes' ) );
```

Comment:
Hooked into this action, the survey note will not get removed from the inbox until a full page reload. Is this what was intended?

If the note should be removed from UI after actioned, perhaps there should be a new option added when actions are registered to specify that a note should be deleted after an action is taken?

Reply:
Hi @mattsherman, thank you for your review.

>Hooked into this action, the survey note will not get removed from the inbox until a full page reload. Is this what was intended?

Yes, I tried to keep the survey note, after the survey has been answered, visible with the message "Thank you for your feedback" until the next full page reload.
Maybe (just to confirm) we can ask @pmcpinto if the survey note should disappear immediately after it was activated or should stay with the message "Thank you for your feedback" until the page reload (like now).